### PR TITLE
[PIN-67] Jetpack navbar

### PR DESCRIPTION
**Pull request by mxj220**
- Converted existing navbar to Jetpack Components
- Implemented NavigationUI, meaning connecting the remaining nav buttons to fragments is simple
- Converted Profile Page to fragment

### CODE REVIEW

File: app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java [hunk 1]
```diff
@@ -3,57 +3,52 @@
...
-        navBar.setSelectedItemId(R.id.navbarEmpty);
         // Check if user is not signed in, if so start auth flow
         if (firebase.getUser() == null) {
-            firebase.launchAuth(this);
+            //firebase.launchAuth(this);
         } else {
             // User is logged in!
             // Log user out for testing purposes
-            firebase.logout(this);
+            //firebase.logout(this);
```
Comment (ckleinhans) [inline app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java:51]: Can we uncomment this or is it supposed to stay commented?

**Review by ckleinhans — CHANGES_REQUESTED** [review]
A few minor things to look at. See comments.

File: app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java (hunk 1 above)
Comment (mxj220) [reply app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java:51]: This was for debugging purposes. I'll uncomment them

File: app/src/main/java/edu/wisc/ece/pinpoint/fragments/ProfilePageFragment.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+            @Override
+            public void onTabUnselected(TabLayout.Tab tab) {
+
+            }
+
+            @Override
+            public void onTabReselected(TabLayout.Tab tab) {
+
+            }
```
[the remark this answers is not in the document]
Comment (mxj220) [reply app/src/main/java/edu/wisc/ece/pinpoint/fragments/ProfilePageFragment.java:59]: Unfortunately they can't be deleted

**Review by ckleinhans — APPROVED** [review]

**Pull request merged**